products/template/tests/test_build.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from dcpy.utils.postgres import get_schema_tables
+from .. import BUILD_SCHEMA
+
+
+# test build stage Load
+@pytest.mark.end_to_end()
+def test_load_db_tables():
+    expected_source_tables = [
+        "dcp_cb2020_wi",
+        "dcp_cdboundaries_wi",
+        "dcp_councildistricts_wi",
+        "dcp_facilities",
+        "dcp_pluto",
+        "dof_shoreline",
+        "dpr_greenthumb",
+        "lpc_historic_districts",
+        "lpc_landmarks",
+    ]
+    built_in_tables = [
+        "spatial_ref_sys",
+        "geography_columns",
+        "geometry_columns",
+    ]
+    all_build_db_tables = sorted(get_schema_tables(BUILD_SCHEMA))
```

Comment:
Should we just filter out the built-in tables from `get_schema_tables`? We'd never want to reference them, I assume. Looks like there's just one other usage of `get_schema_tables`, so no risk.